core-processor/src/executor.rs

```diff
@@ -250,11 +254,33 @@ pub fn execute_wasm<A: ProcessorExt + EnvExt + IntoExtInfo + 'static, E: Environ
         }
     };
 
+    let is_initial = program.get_allocations().is_empty();
+
+    let exports = program.code().exports();
+    if !exports.contains(&dispatch.kind()) {
+        return Ok(DispatchResult {
```

Comment:
I would provide something like `DispatchResult::default(dispatch, pid, allocations)`

Reply:
Agree. Moreover I move this change to the standalone PR siince there are some discussions around memory pages charging.

Reply:
done - #1183